### PR TITLE
doc: add minutes for meeting 25 Sep 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-09-25.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+* src,lib: make ^C print a JS stack trace [#29207](https://github.com/nodejs/node/pull/29207)
+  * Consensus from the 10 TSC members present is opt-in. That seems to align with previous
+    feedback as well as feedback from the tooling team.  
+
+* build: ongoing list of actions for Python 3 compatibility [#25789](https://github.com/nodejs/node/issues/25789)
+  * [ ] tap2junit: v0.1.5 release (nodejs/tap2junit#12) to the Python Packaging Index    [PyPI](https://pypi.org/project/tap2junit)
+    * repo moved to nodejs org, was blocking being able to run jenkins testing. PR
+      to make python 3 compatible.
+  * [ ] node-gyp: release so that npm can pick it up https://github.com/nodejs/node-gyp/issues/1791#issuecomment-533715634
+    * working on doing round trip. Blocked by OSX bug Clauss is working on that.
```
Comment (Trott) [inline meetings/2019-09-25.md]:
```suggestion
    * working on doing round trip. Blocked by OSX bug Christian is working on that.
```

File: meetings/2019-09-25.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  * No concerns from any of the TSC members.
+
+## Strategic Initiatives
+
+* Modules
+  * Moving forward, hit tiny hiccup. Flag caused bootstrap with flag to be different than without
+    flag. Removing flag caused changes and exposed some unexpected regressions
+  * Needed to put proxies in place for built in modules so that they can stay in sync between
+    CommonJS and ES6, but has performance cost. For most modules perf is ok, but for process
+    And buffer, not likely acceptable.  Re-evaluating ways to approach this, in particular with
```
Comment (Trott) [inline meetings/2019-09-25.md]:
```suggestion
    and buffer, not likely acceptable.  Re-evaluating ways to approach this, in particular with
```

File: meetings/2019-09-25.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    performance critical APIs. Pre-req to remove flag
+  * Gut feel is won’t be able to remove flag for 12.x LTS. hoping to land to get things perf
+    consistent.
+
+* N-API
+  * Working on workshop for Node+JS interactive
+  * Work on improving Thread safe functionality in node-addon-api
```
Comment (Trott) [inline meetings/2019-09-25.md]:
```suggestion
  * Working on improving thread-safe functionality in node-addon-api
```

File: meetings/2019-09-25.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+* V8 Currency
+  * Talked about V8 earlier, nothing else to add
+
+* Open Web Standards
+  * Joyee: TPAC last week. Not a lot of Node.js related discussions. 
+  * Want to move core test runner into it’s own package, then vendor in to tools so it can be
+    published on npm.  WebCrypto project under Node.js org wants to use it.  Related to running
+    the webplatform tests.
```
Comment (Trott) [inline meetings/2019-09-25.md]:
```suggestion
    the Web Platform Tests.
```

File: meetings/2019-09-25.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    * repo moved to nodejs org, was blocking being able to run jenkins testing. PR
+      to make python 3 compatible.
+  * [ ] node-gyp: release so that npm can pick it up https://github.com/nodejs/node-gyp/issues/1791#issuecomment-533715634
+    * working on doing round trip. Blocked by OSX bug Clauss is working on that.
+  * [ ] re-vendor npm's gyp back into nodejs/node __/deps/npm/node_modules/node-gyp__
+  * [ ] Travis CI does not test ./configure --ninja test #29415   __*** help needed ***__
+    * Sam, some discussion about testing configure options, not all are tested.  Some tested
+      in docker. Question is why this one?  Less vital if the tap2junit lands.
+  * [ ] node-gyp: Accept Python 3 by default (_semver-major_) nodejs/node-gyp#1844 (macOS fails)
+
```
Comment (sam-github) [inline meetings/2019-09-25.md]:
```suggestion
  * Sam: call-for-volunteers  -- the nodejs/node-gyp team could use more people involved, and that the  https://github.com/refack/GYP3 project does as well. https://github.com/nodejs/node-gyp/issues/1791 has more information. Without more engagement, this situation will slowly get worse.
```

**Comment by mhdawson** [comment]
@sam-github @Trott comments accepted.

**Pull request merged**